specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-07-01-preview/workspaceRP.json

```diff
@@ -4559,6 +4654,181 @@
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "format": {
+          "type": "string"
+        },
+        "isDefaultVersion": {
+          "type": "boolean"
+        },
+        "lifecycleStatus": {
+          "type": "string"
+        },
+        "maxCapacity": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "name": {
+          "type": "string"
+        },
+        "skus": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EndpointModelSkuProperties"
+          }
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/systemData"
+        },
+        "version": {
+          "type": "string"
+        }
+      }
+    },
+    "EndpointModelSkuCapacityProperties": {
+      "type": "object",
+      "properties": {
+        "default": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "maximum": {
+          "format": "int32",
+          "type": "integer"
+        }
+      }
+    },
+    "EndpointModelSkuProperties": {
+      "type": "object",
+      "properties": {
+        "capacity": {
+          "$ref": "#/definitions/EndpointModelSkuCapacityProperties"
+        },
+        "connectionIds": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "deprecationDate": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "deprecationOn": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        },
+        "rateLimits": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EndpointModelSkuRateLimitProperties"
+          }
+        },
+        "usageName": {
+          "type": "string"
+        }
+      }
+    },
+    "EndpointModelSkuRateLimitProperties": {
+      "type": "object",
+      "properties": {
+        "count": {
+          "format": "float",
+          "type": "number"
+        },
+        "renewalPeriod": {
+          "format": "float",
+          "type": "number"
+        },
+        "rules": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EndpointModelSkuRateLimitRuleProperties"
+          }
+        }
+      }
+    },
+    "EndpointModelSkuRateLimitRulePatternProperties": {
+      "type": "object",
+      "properties": {
+        "method": {
+          "type": "string"
+        },
+        "path": {
+          "type": "string"
+        }
+      }
+    },
+    "EndpointModelSkuRateLimitRuleProperties": {
+      "type": "object",
+      "properties": {
+        "count": {
+          "format": "float",
+          "type": "number"
+        },
+        "dynamicThrottlingEnabled": {
+          "type": "boolean"
+        },
+        "key": {
```

Comment:
>key [](http://example.com/codeflow?start=9&length=3)

what format will this property have?

Reply:
https://github.com/Azure/azure-rest-api-specs/blob/2e5be0e72597c6fc8d438f20e38087d900c16427/specification/cognitiveservices/resource-manager/Microsoft.CognitiveServices/preview/2024-06-01-preview/cognitiveservices.json#L4335

we provide the same schema experience based on this schema defined in cognitive service, we don't know the expected value here as we are working as a proxy in this case.